### PR TITLE
Render File_set page and add dependencies for Analytics

### PR DESCRIPTION
**Pull request by njaffer**
Related to #1388

### CODE REVIEW

File: =.bundle/ruby/2.2.0/bundler/gems/curation_concerns-96071bbf6aaf [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 96071bbf6aaf06098fbd75a55905066550cf5ea0
```
Comment (jcoyne) [inline =.bundle/ruby/2.2.0/bundler/gems/curation_concerns-96071bbf6aaf]: This should not be included.

File: =.bundle/ruby/2.2.0/bundler/gems/kaminari-7f7108e12bdd [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit 7f7108e12bdd64eb5a76177633768feb14df5453
```
Comment (jcoyne) [inline =.bundle/ruby/2.2.0/bundler/gems/kaminari-7f7108e12bdd]: This should not be included.

File: lib/sufia/engine.rb [hunk 3]
```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```
Comment (jcoyne) [inline lib/sufia/engine.rb]: These should be autoloaded an not required in an initializer.

Comment (njaffer) [inline lib/sufia/engine.rb]: @jcoyne but it does not autoload and without this the Legato fails and gives an error " undefined method sufia__download". If I take out this line require 'sufia/analytics' only, even then I get an error. I have compared this with Sufia 6 and this is what was added in one of the PRs. Yesterday we looked at this with @mjgiarlo

**Pull request closed without merging**